acme-client.gemspec

```diff
@@ -26,5 +26,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'vcr', '~> 2.9'
   spec.add_development_dependency 'webmock', '~> 3.8'
 
-  spec.add_runtime_dependency 'faraday', '~> 0.17', '< 1.0.0'
```

Comment:
I can't exactly recall why we added this strictness but if the tests pass I would rather just drop `'< 1.0.0'` and open up to `'>= 0.17' `.